google-cloud-spanner/src/main/java/com/google/cloud/spanner/MetricRegistryConstants.java

```diff
@@ -45,9 +45,13 @@
   static final String MAX_IN_USE_SESSIONS = "cloud.google.com/java/spanner/max_in_use_session";
   static final String MAX_ALLOWED_SESSIONS = "cloud.google.com/java/spanner/max_allowed_sessions";
   static final String IN_USE_SESSIONS = "cloud.google.com/java/spanner/in_use_sessions";
+  static final String GET_SESSION_TIMEOUT = "cloud.google.com/java/spanner/get_sessions_timeout";
+
   static final String MAX_IN_USE_SESSIONS_DESCRIPTION =
       "The maximum number of sessions in use during the last 10 minute interval.";
   static final String MAX_ALLOWED_SESSIONS_DESCRIPTION =
       "The maximum number of sessions allowed. Configurable by the user.";
   static final String IN_USE_SESSIONS_DESCRIPTION = "The number of sessions currently in use.";
+  static final String SESSIONS_TIMEOUT_DESCRIPTION =
+      "The number of get session timeouts due to pool exhaustion";
```

Comment:
Based on previous comment this could be changed to
`The number of retries to open a session. Retries occur when session pool is exhausted.`